src/ui/pages/Invoice/InvoicePage.css

```diff
@@ -19,22 +19,4 @@
     color:red;
 }
 
-.calendar {
```

Comment:
My point is there is already class for calendar. Somewhere else and you are overwriting it. If you want a different styles, you need to make a new class for it.

Reply:
I will leave the purchase as it is, since you also have the billing party autocomplete to also fix. so pleae fix this when you do that, you also need this for sales, i assume (if you have time to code it ofc)

Reply:
I see the problem here, its due to height being in dvh, i have changed it back to px and it works .